### PR TITLE
Replacing instances of "moxie" with "nolte"

### PR DESCRIPTION
**Pull request by btchae**
Let me know if I missed anything.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -199,8 +199,8 @@ <h3 class="features__title">We like blocks, elements and modifiers.</h3>
     <div class="footer__copyright">
       Leean is under <a rel="nofollow" href="https://www.gnu.org/licenses/gpl-2.0.txt">GPL</a> license.
     </div>
-    <div class="footer__brand--moxie">
-      <img src="src/images/moxie-footer.png" alt="" class="footer_brand__media">
+    <div class="footer__brand--nolte">
+      <img src="src/images/nolte-footer.png" alt="" class="footer_brand__media">
```
Comment (mitogh) [inline index.html:203]: Did you renamed the files as well?

**Review by mitogh — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (btchae) [reply index.html:203]: I did not. They have the moxie logo so I think there would have to be new files.

Comment (mitogh) [reply index.html:203]: Okay can you add those then?

Thanks.

Comment (btchae) [reply index.html:203]: I removed moxie-footer.png with nolte-footer.png

**Review by mitogh — APPROVED** [review]

**Pull request merged**